CartingService/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# this needs to be run from the root of the repository
```

Comment:
Could you please, improve performance of docker files?

Reply:
Fixed